src/Document.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace OpenFoodFacts;
+
+use OpenFoodFacts\Document\FoodProduct;
```

Comment:
these 3 imports are unused, please remove it

Reply:
I will remove this imports.